core/src/main/java/com/datastrato/gravitino/connector/PropertiesMetadata.java

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Strangely, it's a new file, not one that was renamed.

Reply:
If the file is not just only renamed, but also has a lot of changes. Git will treat it as a new file with old file deleted.